### PR TITLE
Static object changes

### PR DESCRIPTION
**Pull request by PhysicsJon**

### CODE REVIEW

**Review by bjss-andrew — DISMISSED** [review]

File: src/NHSDPublicBrowseAcceptanceTests.Actions/Interactions.cs [hunk 1]
```diff
@@ -20,9 +19,6 @@ public Interactions(IWebDriver driver)
             // Initialize a WebDriverWait that can be reutilized by all that inherit from this class
             wait = new WebDriverWait(new SystemClock(), driver, TimeSpan.FromSeconds(10),
                 TimeSpan.FromMilliseconds(100));
-
-            // Initialize the page objects
-            pages = new PageObjects().Pages;
         }
     }
 }
```
Comment (bjss-chris) [inline src/NHSDPublicBrowseAcceptanceTests.Actions/Interactions.cs]: Just missing a trailing newline.

File: src/NHSDPublicBrowseAcceptanceTests.Actions/Pages/BuyersGuide.cs [hunk 2]
```diff
@@ -11,22 +11,22 @@ public BuyersGuide(IWebDriver driver) : base(driver)
...
+            return driver.FindElement(Objects.Pages.BuyersGuide.CatalogueSolutionGuide).Displayed;
         }
 
         public bool ServiceDeskGuidanceContentDisplayed()
         {
-            return driver.FindElement(pages.BuyersGuide.ServiceDeskGuide).Displayed;
+            return driver.FindElement(Objects.Pages.BuyersGuide.ServiceDeskGuide).Displayed;
         }
     }
 }
```
Comment (bjss-chris) [inline src/NHSDPublicBrowseAcceptanceTests.Actions/Pages/BuyersGuide.cs]: Just missing a trailing newline.

File: src/NHSDPublicBrowseAcceptanceTests.Actions/Pages/Header.cs [hunk 3]
```diff
@@ -11,17 +11,17 @@ public Header(IWebDriver driver) : base(driver)
...
+            driver.FindElement(Objects.Pages.Header.Container).Displayed.Should().BeTrue();
         }
 
         public void BannerDisplayed()
         {
-            driver.FindElement(pages.Header.Banner).Displayed.Should().BeTrue();
+            driver.FindElement(Objects.Pages.Header.Banner).Displayed.Should().BeTrue();
         }
     }
 }
```
Comment (bjss-chris) [inline src/NHSDPublicBrowseAcceptanceTests.Actions/Pages/Header.cs]: Just missing a trailing newline.

**Review by bjss-chris — CHANGES_REQUESTED** [review]
Just a couple of EOF issues to fix but looks fine otherwise.

**Review by bjss-chris — APPROVED** [review]

**Pull request merged**